examples/prom/elliptic_eigenproblem_global_rom.cpp

```diff
@@ -309,7 +312,7 @@ int main(int argc, char *argv[])
     if (merge)
     {
         mergeTimer.Start();
-        options = new CAROM::Options(fespace->GetTrueVSize(), max_num_snapshots, nev,
+        options = new CAROM::Options(fespace->GetTrueVSize(), max_num_snapshots,
```

Comment:
Should `max_num_snapshots` be an input argument?

Reply:
That's a good question. Many, if not all, of the MFEM examples hardcoded the ROM options, e.g., 
```
    int max_num_snapshots = 100;
    bool update_right_SV = false;
    bool isIncremental = false;
```
We can try to make them input arguments in a separate PR.